client/webserver/site/src/css/market.scss

```diff
@@ -420,6 +420,47 @@ table.balance-table button:hover {
   td {
     padding: 2px 10px;
     white-space: nowrap;
+    flex: 3;
+
+    &.last {
+      flex: 1;
```

Comment:
It wasn't clear why you were using flex here now, so I went through and removed the flex and it still worked fine. Was there another reason to use flex that I'm not seeing?

https://github.com/decred/dcrdex/commit/beb9ed2496449163b48abcfb63602536fa46c230

The one thing I did notice is that the arrow icon seems to be off center, so I loaded the svg into icomoon and got this

![image](https://user-images.githubusercontent.com/6109680/107811301-91189880-6d33-11eb-91a6-0fccccfc23fd.png)
which seems like a big clue that something was wrong with your icon. I would start by using an svg editor to make a square icon with the arrow centered vertically.